### DISCUSSION TITLE
Anyone else having trouble on macOS with restoring sessions and unsaved files with 147.x?

### DISCUSSION THREAD

**Discussion by ylluminate** [zed-industries/zed#15698] [Feature Requests]
Seems like the new updates to restore sessions, including unsaved/dirty buffers, is not yet functional on macOS with 147.x (0.147.2 specifically)... even though it appears it should be from the update info and using the most recent `zed@preview` version from Homebrew.

Anyone else see this or resolve this?

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
